lib/trino-parquet/src/test/java/io/trino/parquet/reader/TestParquetReader.java

```diff
@@ -37,7 +37,7 @@
 import static io.trino.spi.type.IntegerType.INTEGER;
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class TestParquetReaderMemoryUsage
+public class TestParquetReader
```

Comment:
we already have a test with this name (`io.trino.plugin.hive.parquet.TestParquetReader`)
would it make sense to pick something else?

or maybe add the new test to that other test class? or to a new class?

Reply:
renamed `io.trino.plugin.hive.parquet.TestParquetReader` to `io.trino.plugin.hive.parquet.TestQuickParquetReader`